### DISCUSSION TITLE
LG CX doesn't turn off when PC shuts down

### DISCUSSION THREAD

**Discussion by xreyuk** [JPersson77/LGTVCompanion#192] [Q&A] [answered]
Hi All, 

At present my LG TV won't turn off when I shutdown my PC (it does wake successfully from off/sleep). It used to but I can't remember when it stopped working.

I have the 'restart' box checked in the settings and the program is managing the device. Any ideas what settings I can try change to get this working?

**Comment by JPersson77** [comment, marked as answer]
Hi @xreyuk you could try to change the shutdown timing in the device configuration. It is also a possibility that the IP of you TV has changed

**Reply by xreyuk** [reply]
Thanks. I changed it to early and it seems to work better.
